package.json

```diff
@@ -6,6 +6,6 @@
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1"
   },
-  "version": "0.1.7",
+  "version": "0.1.8",
```

Comment:
You should not be editing this **ever**. This should be handled by `npm version` itself.

Reply:
reverted back to original

Reply:
👍